### DISCUSSION TITLE
Management Change and Project Restructuring

### DISCUSSION THREAD

**Discussion by AlexandraFuentes** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#150] [General]
Hey @uprm-inso4101-2024-2025-s1/campus-resource-and-event-management,

We wanted to update you all on an important change in our project management. After discussing with the team and in agreement with the professor and Lex, old managers have decided to step down as project managers.

@Adrianortiz-PR and I (@AlexandraFuentes) will be taking over the management roles. We’re looking forward to working more closely with each of you to ensure that the project moves forward smoothly and efficiently.

There will be further restructuring to streamline tasks and ensure clear communication moving forward. We’ll be discussing this in our scrum meeting today (9/26) soon to go over these changes and discuss how we can best support you in reaching the milestone goals.

@lex939 @MarkoSchuetz  — tagging you for visibility on this change and to keep you updated.

Feel free to reach out if you have any questions or concerns, and thank you for your continued hard work.

**Edit:**

Additionally, we’ve made some adjustments:
- **Edzel** (@EdzelOrtiz) will be the new team leader for the **Profile Tuner Team**.
- The **Login and User team** will be merged into a **User Team**, which will focus on user management across the project, and **Pierre** (@pierreg-vales) will lead this new combined team. 
- **Angel** (@AngelUPR) will be integrated into the Profile Tuner team.
- **Eduardo** (@SaucyRexy) will be our Documentation lead.

---

### **Summary of Feature Teams and Team Leads:**

- **User Team (Login & User Features)**: Led by Pierre
  - *Goal for this sprint and milestone:* Finalize user authentication and profile management.
  
- **Profile Tuner**: Led by Edzel
  - *Goal for this sprint:* Frontend of webpage 
  - *Goal for this milestone:* Front end and define the backend student classes with their tags 

- **Calendar**: Led by Ojani
  - *Goal for this sprint:* Frontend of webpage 
  - *Goal for this milestone:* Defining event class
  
- **Homepage**: Led by Jean
  - *Goal for this sprint:* Frontend of the webpage
  -  *Goal for this milestone:* Integrate front end with backend 
---

### **Overall Goal for this Sprint:**

- Frontend of these initial features

### **Overall Goal for the Milestone (Milestone 2):**

- Define classes, integrate the frontend and backend. Ensure that most initial features are working together and tested thoroughly for stability.

**Comment by MarkoSchuetz** [comment]
This looks like you have come to a good solution. This will affect several things for all the students involved in the project and we have to be sure to reflect them accordingly. One thing that I can see right away and which I will have to take care of, is the setup of the Teammates questionnaires. For the first milestone everything will remain as is. But for the subsequent milestones I will have to create new groups since in the managers for the developer-to-manager evaluations will be different. Also the managers for the manager-to-manager evaluations will be different. You will therefore receive emails from Teammates that reflect these changes.

**Comment by lex939** [comment]
I agree with the professor. I consider this a good solution.

However, from what I understand, @AlexandraFuentes and @Adrianortiz-PR were both Team Leaders before this restructuring. Please specify who are the two project members that will now become Team Leaders due to this change. Please also specify any other changes that will happen due to this change on this discussion.

**Reply by AlexandraFuentes** [reply]
Addressed in the new edit!

**Reply by lex939** [reply]
Read the new edit. Thank you for specifying.
